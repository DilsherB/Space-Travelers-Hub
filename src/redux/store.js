import { configureStore } from "@reduxjs/toolkit";
import missionReducer from "./missions/missionsSlice";
import { reducer as rocketReducer } from "./rockets/rocketsSlice";

const store = configureStore({
  reducer: {
    missions: missionReducer,
    rockets: rocketReducer
  },
});
export default store;
