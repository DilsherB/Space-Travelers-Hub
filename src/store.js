import { configureStore } from "@reduxjs/toolkit";
import missionReducer from "./features/missions/missionsAPI";

const store = configureStore({
  reducer: {
    missions: missionReducer,
  },
});
export default store;
