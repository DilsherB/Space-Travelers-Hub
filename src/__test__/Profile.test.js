import { render } from "@testing-library/react";
import React from "react";
import { Provider } from "react-redux";
import Myprofile from "../routes/MyProfile";
import store from "../redux/store";

describe("Test Profile components", () => {
  it("renders correctly", () => {
    const { container } = render(
      <Provider store={store}>
        <Myprofile />
      </Provider>
    );
    expect(container).toMatchSnapshot();
  });
});
