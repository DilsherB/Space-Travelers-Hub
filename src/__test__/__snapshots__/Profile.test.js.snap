// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Profile components renders correctly 1`] = `
<div>
  <div
    class="row"
  >
    <div
      class="col"
    >
      <h2>
        Mission
      </h2>
      <div
        class="list-group"
      >
        <div
          class="list-group-item"
        >
          No reservation
        </div>
      </div>
    </div>
    <div
      class="col"
    >
      <h2>
        Rockets
      </h2>
      <div
        class="list-group"
      >
        <div
          class="list-group-item"
        >
          No reservation
        </div>
      </div>
    </div>
  </div>
</div>
`;
